### PR TITLE
Set mariadb version to 10.4.3 to solve barracuda issues on Totara install

### PR DESCRIPTION
**Pull request by barryo-learningpool**

### CODE REVIEW

**Comment by derschatta** [comment]
Hi @barryo-learningpool can you add some details on the requirements and where 10.3 does not meet it?

mariadb 10.4 is not the latest stable version. I personally would prefer to use the latest stable version if there's no hard reason not to.

**Comment by derschatta** [comment]
@barryo-learningpool would you be able to bump the version up to 10.4.6 which is now the latest stable version?

**Comment by barryo-learningpool** [comment]
Hi @derschatta , sure no problem. I've done that now. I saw the barracuda issues when I first started using this with a cli install of Totara. I didn't take a note of exactly what it said at the time but this fixed it for me.

File: compose/mariadb.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "2"
 services:
 
   mariadb:
-    image: mariadb:10
+    image: mariadb:10.4.6
```
Comment (derschatta) [inline compose/mariadb.yml]: I just realised that 10 actually would automatically give you the latest 10.4.x version. We have several options:
* Leave it and always get the newest version
* Tie the version down to 10.4
* Tie the version down to 10.4.7 (they already released a new minor version)

I would go for the second option. What do you think, @barryo-learningpool ?

**Comment by derschatta** [comment]
haven't heard back. created new pr #64

**Pull request closed without merging**